### DISCUSSION TITLE
Setting shared IP query

### DISCUSSION THREAD

**Discussion by mlcl-thomascommons** [splunk/splunk-connect-for-snmp#351] [Q&A] [answered]
Hi team!

Hoping that someone can assist. I am pretty much through the deployment of SC4SNMP but I am running into a hurdle which I think it because I am not defining the shared IP in the values.yaml file.

Looking at the documents it says to replace X.X.X.X with the shared IP (I assume this in the actual VM IP address), but there is no mention of this in the actual yaml file.

Not sure if I missed something. 

Appreciate any assistance.

Regards,
Tom

**Comment by omrozowicz-splunk** [comment, marked as answer]
Hello Tom,
Not sure if you're still looking for the answer to this question, but you helped us to identify inconsistency in our documentation.

That shared IP X.X.X.X is the IP address of the trap's target (right, this should be your VM IP address) and it should be configured here:

```
traps:
  loadBalancerIP: 10.1.0.1
```

We need to fix the documentation to make it look like this:

```
traps:
  loadBalancerIP: X.X.X.X
```

Creating issue out of this one. Thanks.
Olga
